python/pyspark/sql/dataframe.py

```diff
@@ -1393,6 +1393,16 @@ def replace(self, to_replace, value=None, subset=None):
         |null|  null| null|
         +----+------+-----+
 
+        >>> df4.na.replace('Alice', None).show()
```

Comment:
I assume we removed `dataframe.replace` to promote use `dataframe.na.replace`? The doc says they are aliases anyway. I don't know but I tend to agree with paring doc tests and this looks removed in https://github.com/apache/spark/commit/ff26767c03cc76e7e86b238300367fa0d9b3e6a4.

Let's leave this as is for now. I don't want to make this PR complicated.